authors/apps/authentication/renderers.py

```diff
@@ -21,6 +24,23 @@ def render(self, data, media_type=None, renderer_context=None):
             rendering errors.
             """
             return super(UserJSONRenderer, self).render(data)
+        try:
+            """
+            Checks whether the user is active and returns a message
+            incase the user is not active.
+            If the user is active, then it will pass.
+            """
+            confirm_user = data['email']
+            user = User.objects.get(email=data['email'])
+            if user.is_active is False:
+                return json.dumps({
+                    "Message":
+                    "Account is not active. If registering, "
+                    "kindly click the link sent to your email to "
+                    "complete registration."
+                    })
+        except KeyError:
+            pass
```

Comment:
@Bileonaire From the Zen of Python `Errors should never pass silently.` let's stick to this.